weave/index/index_pandas.py

```diff
@@ -134,8 +138,11 @@ def sync_index(self):
             self.generate_index()
             return
         if len(index_paths) > 20:
-            warnings.warn(f"The index basket count is {len(index_paths)}. " +
-                 "Consider running weave.IndexPandas.clean_up_indices")
+            if self.auto_cleanup:
```

Comment:
I'm assuming that auto_cleanup statement should be the default, but when I ran it for the first time with more than 20 buckets added, it skipped this statement and went straight to the user warning. Is that expected?

Reply:
My mistake, it does work as expected. I was on the wrong branch